taxi/views.py

```diff
@@ -81,9 +114,10 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
-class DriverListView(LoginRequiredMixin, generic.ListView):
+class DriverListView(LoginRequiredMixin, SearchableListView):
     model = Driver
     paginate_by = 5
+    search_field = "username"
```

Comment:
The `search_field` attribute is set to 'username', which is appropriate for the `DriverListView` since it likely corresponds to a field in the `Driver` model.